framework/auth/core.py

```diff
@@ -1385,7 +1386,10 @@ class Institution(StoredObject):
 
     _id = fields.StringField(index=True, unique=True, primary=True)
     name = fields.StringField(required=True)
+
     logo_name = fields.StringField(required=True)
+    domains = fields.StringField(list=True)
+    metadata_request_fields = fields.DictionaryField()
```

Comment:
This field is to store where in the request body are we expecting the metadata to be stored, since that will vary depending on the institution. It might not end up being a dictionary field.